content/blog/nats-server-2.9.17-release.md

```diff
@@ -0,0 +1,124 @@
++++
+date = "2023-05-19"
+draft = false
+title = "NATS Server 2.9.17 Release"
+author = "Byron Ruth"
+categories = ["Engineering"]
+tags = ["NATS", "Release"]
++++
+
+The NATS maintainers are proud to announce the [2.9.17 release](https://github.com/nats-io/nats-server/releases/tag/v2.9.17)! We want to thank all of the people who contributed to this release! If **you** are interested in contributing, please check out [all the ways you can](https://nats.io/contributing/)!
+
+The key areas this post will cover include:
+
+- [WebSocket transport truncation](#-websocket-transport-truncation)
+- [Health monitor improvements](#-health-monitor-improvements)
+- [Leafnode fleet optimizations](#-leafnode-fleet-optimizations)
+- [Monotonic time calculations](#-monotonic-time-calculations)
+
+For the entirety of the improvements and fixes, check out the [release notes](https://github.com/nats-io/nats-server/releases/tag/v2.9.17).
+
+## 🐘 WebSocket transport truncation
+
+As part of the [P99 peformance work in the 2.9.16 release](https://nats.io/blog/nats-server-2.9.16-release/#-p99-performance), a regression was introduced, resulting in [faulty subscription behavior in clients](https://github.com/nats-io/nats.ws/issues/194) received fairly large messages over the WebSocket transport. Internally this had to do with prematurely deallocating a buffer to a shared pool before data was entirely flushed on a connection.
+
+However, the upside of addressing this issue resulted in more (correct) optimizations for the WebSocket transport, including reducing the overall memory footprint and switching to a [faster compression algorithm](https://github.com/nats-io/nats-server/pull/4087).
```

Comment:
Incompatible tense changes with "resulted" there. Suggest: "... this issue is more ..."